torchvision/prototype/datasets/_builtin/country211.py

```diff
@@ -0,0 +1,51 @@
+import functools
+import pathlib
+from typing import Any, Dict, List, Tuple
+
+from torchdata.datapipes.iter import IterDataPipe, Mapper, Filter
+from torchvision.prototype.datasets.utils import Dataset, DatasetConfig, DatasetInfo, HttpResource, OnlineResource
+from torchvision.prototype.datasets.utils._internal import hint_sharding, hint_shuffling
+from torchvision.prototype.features import EncodedImage, Label
+
+
+class Country211(Dataset):
+    def _make_info(self) -> DatasetInfo:
+        return DatasetInfo(
+            "country211",
+            homepage="https://github.com/openai/CLIP/blob/main/data/country211.md",
+            valid_options=dict(split=("train", "valid", "test")),
```

Comment:
For consistency with other datasets.

```suggestion
            valid_options=dict(split=("train", "val", "test")),
```

With that we probably need a mapping between the split and the folder name.